autopilot/contractor.go

```diff
@@ -218,7 +220,8 @@ func (c *contractor) performContractMaintenance(ctx context.Context, w Worker) (
 	c.mu.Unlock()
 
 	// run checks
-	updatedSet, toArchive, toStopUsing, toRefresh, toRenew, err := c.runContractChecks(ctx, w, contracts, minScore)
+	maxKeepLeeway := addLeeway(state.cfg.Contracts.Amount, 1-leewayPctRequiredContracts)
```

Comment:
Can't we get this within `c.runContractChecks`? Seems like we have access to `ctx, w, contracts, minScore` and we can get `state` from `c`.